dex/market.go

```diff
@@ -9,6 +9,12 @@ import (
 	"strings"
 )
 
+const (
+	// defaultEpochDuration is the spec defined default markets epoch duration
+	// in milliseconds.
+	defaultEpochDuration uint64 = 60000
```

Comment:
Turns out that 60 sec is an extremely long epoch.  I think we should revise the spec to make it 10 sec, 20 tops.

Reply:
> I think we should revise the spec to make it 10 sec, 20 tops.

20 sec should be good.